### ISSUE TITLE
GO TO MESSAGE link broken

### ISSUE THREAD

**Issue by antgel** [RocketChat/Rocket.Chat#10926]
### Description:

As per title.

### Server Setup Information:

* Version of Rocket.Chat Server: 0.64.2. Also failed on today's `develop`.
* Operating System: Linux
* Deployment Method(snap/docker/tar/etc): Docker / local
* Number of Running Instances: 1
* DB Replicaset Oplog: Disabled
* Node Version: 8.11.1
* mongoDB Version: Whatever is in Docker

### Steps to Reproduce:

1. Send a message to an offline user who has email notifications configured.

### Expected behavior:

User should receive an email and GO TO MESSAGE link should point to the relevant message or channel.

### Actual behavior:

The link is cut off. e.g. http://localhost:3000/direct instead of http://localhost:3000/direct/some.user, or http://localhost:3000/channel instead of http://localhost:3000/channel/funstuff. This confuses users who don't have Rocket.Chat open all the time (shame ;) ).

### Relevant logs:

Here's the HTML from the email as seen in [MailHog](https://github.com/mailhog/MailHog), in which we see that the link is indeed broken:
```
<html><table border=3D"0" cellspacing=3D"0" cellpadding=3D"0" =
width=3D"100%" bgcolor=3D"#f3f3f3" style=3D"color:#4a4a4a;font-family: =
Helvetica,Arial,sans-serif;font-size:14px;line-height:20px;border-collapse:=
collapse;border-spacing:0;margin:0 auto"><tr><td style=3D"padding:1em"><tab=
le border=3D"0" cellspacing=3D"0" cellpadding=3D"0" align=3D"center" =
width=3D"100%" style=3D"width:100%;margin:0 auto;max-width:800px"><tr><td =
bgcolor=3D"#ffffff" style=3D"background-color:#ffffff; border: 1px solid =
#DDD; font-size: 10pt; font-family: Helvetica,Arial,sans-serif;"><table =
width=3D"100%" border=3D"0" cellspacing=3D"0" cellpadding=3D"0"><tr><td =
style=3D"background-color: #04436a;"><h1 style=3D"font-family: Helvetica,=
Arial,sans-serif; padding: 0 1em; margin: 0; line-height: 70px; color: =
#FFF;">Rocket.Chat</h1></td></tr><tr><td style=3D"padding: 1em; font-size: =
10pt; font-family: Helvetica,Arial,sans-serif;"><strong>antony.gelberg.=
private</strong> sent a message on <strong>#general</strong>:<br/><br/>@ant=
gel I&#39;m testing<hr style=3D"margin: 20px auto; border: none; =
border-bottom: 1px solid #dddddd;"><p style=3D"text-align:center;margin-bot=
tom:8px;"><a style=3D"color: #fff; padding: 9px 12px; border-radius: 4px; =
background-color: #04436a; text-decoration: none;" href=3D"http://localhost=
:3000/channel">GO TO MESSAGE</a></td></tr></table></td></tr><tr><td =
border=3D"0" cellspacing=3D"0" cellpadding=3D"0" width=3D"100%" =
style=3D"font-family: Helvetica,Arial,sans-serif; max-width: 800px; margin:=
 0 auto; padding: 1.5em; text-align: center; font-size: 8pt; color: =
#999;">Powered by <a href=3D"https://rocket.chat" target=3D"_blank">Rocket.=
Chat</a></td></tr></table></td></tr></table></html>
```

I'm happy to provide more info if required.

Cc @sampaiodiego I know you were working on notifications, maybe interesting for you?
Cc @schurder You might want to follow this issue.

**Comment by kaiiiiiiiii** [comment]
Duplicate of #10897. Fixed in PR #10911

**Comment by antgel** [comment]
Thanks, I did check for existing bugs, but some hours passed and #10897 was opened in the meantime.:)
